packages/api/src/schema/db-schema.yaml

```diff
@@ -1018,6 +1019,39 @@ components:
           description:
             Timestamp (in milliseconds) at which the asset was deleted
           example: 1587667174725
+          index: true
```

Comment:
I don't think we need all these indexes, let's chat about it to minimize how many we create here.